src/index.ts

```diff
@@ -14,16 +15,28 @@ declare global {
   }
 }
 
-export default function toBeAValidHAR(this: jest.MatcherUtils, har: Har): { message: () => string; pass: boolean } {
+export default function toBeAValidHAR(
+  this: jest.MatcherUtils | MatcherState,
+  har: Har,
+): { message: () => string; pass: boolean } {
+  const request = har?.log?.entries?.[0]?.request || undefined;
+  if (!request) {
+    return {
+      message: () => 'expected supplied HAR to be valid\n\nUnable to destructure `log.entries[0].request`',
+      pass: false,
+    };
+  }
```

Comment:
Added this logic (as well as additional tests) so passing junk like empty objects fail properly.